models/contracts/optimism/contracts_optimism_contract_overrides.sql

```diff
@@ -0,0 +1,21 @@
+{{ 
+  config(
+    schema = 'contracts_optimism', 
+    alias='contract_overrides',
+    post_hook='{{ expose_spells(\'["optimism"]\',
+                              "sector",
+                              "contracts",
+                              \'["msilb7", "chuxinh"]\') }}'
+    ) 
+}}
+
+select 
+  contract_address
```

Comment:
added a comment before review - but we could make this "lower() AS contract_address"